src/pages/styles.tsx

```diff
@@ -0,0 +1,24 @@
+import { styled } from "@mui/system";
+import { Box } from "@mui/material";
+
+export const StyledModuleBox = styled(Box)(({ theme }) => ({
```

Comment:
Keep the style specific to the component and pages

pages/Dashboard/index.tsx
pages/Dashboard/style.tsx

Reply:
Did that dai!